dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -86,11 +87,20 @@ const realTimeConfig = (
 		'gdpr_consent=CONSENT_STRING',
 	].join('&');
 
+	const urls = [
+		usePrebid ? prebidURL : '',
+		usePermutive ? permutiveURL : '',
+	].filter((url) => url);
```

Comment:
```suggestion
	].filter(Boolean); // remove empty strings, which are falsey
```

Reply:
+1 for the comment - it wasn't at all clear to me that that was the intent.

Reply:
This also confused me! Good suggestion